Source/WebCore/rendering/RenderSlider.cpp

```diff
@@ -60,7 +60,7 @@ RenderSlider::RenderSlider(HTMLInputElement& element, RenderStyle&& style)
 
 RenderSlider::~RenderSlider() = default;
 
-HTMLInputElement& RenderSlider::element() const
+CheckedRef<HTMLInputElement> RenderSlider::element() const
```

Comment:
Usually we create a separate version called `checkedElement()` rather than changing the normal one.

(Also any reason this is not return a `Ref`? Elements can be ref-counted)